controllers/base_controller.ts

```diff
@@ -76,8 +81,12 @@ class BaseController<T> {
 
         if (Mongoose.prototype.isValidObjectId(itemIdToDelete)) {
             try {
-                await this.model.findByIdAndDelete(itemIdToDelete);
-                res.status(200).send();
+                const result = await this.model.findByIdAndDelete(itemIdToDelete);
+                if (result !== null) {
```

Comment:
```suggestion
                if (result) {
```
I think it does the same thing